docs/pages/enroll-resources/machine-id/workload-identity/workload-attestation.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: Workload Attestation
+description: An overview of the Workload Attestation functionality
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. This means that some
+features may be missing. We're actively looking for design partners to help us
+shape the future of Workload Identity and would love to
+[hear your feedback](mailto:product@goteleport.com).
+</Admonition>
+
+Workload Attestation is the process of verifying the identity of a workload that
+has connected to the Workload API. Workload attestors resolve the process ID
+of the workload against platform-specific APIs to determine more information
+(known as attestation metadata) about the workload. This information can then
+be used to make decisions about which SPIFFE IDs should be issued to the
+workload.
+
```

Comment:
I think it would help me understand this to explain the architecture of Workload Attestation before we describe individual attestors. Where do the attestors run in relationship to a Teleport cluster, `tbot`, and workloads?

Reply:
b29809fefd